lua/lvim/core/treesitter.lua

```diff
@@ -89,6 +124,13 @@ M.setup = function()
 
   treesitter_configs.setup(opts)
 
+  if vim.opt.foldmethod:get() == "expr" and vim.opt.foldexpr:get() == "nvim_treesitter#foldexpr()" then
+    vim.opt.foldmethod = "manual"
+  end
+
+  vim.g.indent_blankline_use_treesitter = false
+  vim.g.indent_blankline_show_current_context = false
```

Comment:
```suggestion
```

please change it here instead
https://github.com/LunarVim/LunarVim/blob/30c65cfd74756954779f3ea9d232938e642bc07f/lua/lvim/core/indentlines.lua#L23-L24

Reply:
It can't be done in `lvim.builtin.indentlines` because then indent blankline won't register an autocommand on CursorMoved at all and setting `vim.b.indent_blankline_show_current_context = true` won't work. These need to be disabled after `indent_blankline.setup()` is called

Reply:
I meant why not set  `use_treesitter = false` from the get-go instead?

Reply:
Oh ok. Will do that then

Reply:
> Edit: maybe we can use `require("indent_blankline.commands").toggle` directly

CursorMoved is set in setup [here](https://github.com/lukas-reineke/indent-blankline.nvim/blob/db7cbcb40cc00fc5d6074d7569fb37197705e7f6/lua/indent_blankline/init.lua#L131)

Reply:
Relevant issue in indent-blankline: [link](https://github.com/lukas-reineke/indent-blankline.nvim/issues/440)